bytecode/src/bytecode.rs

```diff
@@ -176,9 +177,7 @@ pub enum Instruction {
     JumpIfFalseOrPop {
         target: Label,
     },
-    MakeFunction {
-        flags: FunctionOpArg,
```

Comment:
This surprises me, don't we need a codeflags when creating the function? Or are the flags already packed in the codeobject itself before this happens?

Reply:
They're already in the `CodeObject` that `MakeFunction` is operating on.

Reply:
Interesting, this looks good!